### DISCUSSION TITLE
Version 6.1.0 release

### DISCUSSION THREAD

**Discussion by rockfordlhotka** [MarimerLLC/csla#3031] [Announcements]
CSLA .NET version 6.1.0 adds support for Maui and includes bug fixes and one breaking change.

* [#2549](https://github.com/MarimerLLC/csla/issues/2549) Add UI helpers for Maui
* [#2672](https://github.com/MarimerLLC/csla/issues/2672) Enhance PropertyChanged behavior in `Csla.Xaml.ViewModelBase`
* [#2764](https://github.com/MarimerLLC/csla/issues/2764) Update snippets NuGet installer for VS2022
* [#2922](https://github.com/MarimerLLC/csla/issues/2922) Blazor no longer requires text-based serialization
* [#2946](https://github.com/MarimerLLC/csla/issues/2946) LocalProxy now properly disposes DI scope and dependencies
* [#2953](https://github.com/MarimerLLC/csla/issues/2953) DataAnnotations rules now support DI
* [#2957](https://github.com/MarimerLLC/csla/issues/2957) Child data portal expections now flow up with details
* [#2981](https://github.com/MarimerLLC/csla/issues/2981) You can now get a `System.Reflection.PropertyInfo` from `Csla.Blazor.PropertyInfo`
* [#3007](https://github.com/MarimerLLC/csla/issues/3007) Add new BlazorServerExample project to the `Samples` folder
* [#3014](https://github.com/MarimerLLC/csla/issues/3014) Fix bug in child data portal
* [#3025](https://github.com/MarimerLLC/csla/issues/3025) Fix issue with analyzers

### Change List

* [Changes in this release](https://github.com/MarimerLLC/csla/issues?q=is%3Aclosed+project%3Amarimerllc%2F3+)
* [Breaking changes](https://github.com/MarimerLLC/csla/issues?q=is%3Aissue+is%3Aclosed+project%3Amarimerllc%2F3+label%3A%22flag%2Fbreaking+change%22)

### Dependency Updates

* Grpc.Net.Client
* Grpc.Tools
* Newtonsoft.Json
* RabbitMQ.Client
* Microsoft.NETCore.UniversalWindowsPlatform
* Xamarin.Forms
* Microsoft.AspNet.WebApi.Client
* Microsoft.AspNet.WebPages
* Microsoft.Web.Infrastructure
* Microsoft.AspNet.WebApi.Core
* Microsoft.AspNet.Mvc
* Microsoft.AspNet.Razor
* Google.Protobuf
* Microsoft.CodeAnalysis.CSharp

### Contributors

@joshhanson314
@rockfordlhotka
@TheCakeMonster
@jasonbock

**Comment by Chicagoan2016** [comment]
Can't wait to try the new version! and also glad to see @JasonBock active in Csla community!

**Comment by AbbasMalik** [comment]
Is the projectTracker also updated?

**Reply by joshhanson314** [reply]
No. There is a separate issue for tracking that as #2551

**Comment by ryanaw83** [comment]
In latest CSLA Maui (6.1.0 for iOS/Android), Csla.Xaml.ViewModelBase no longer seems to be accessible (although it can be accessed from Csla.Axml).  Should Csla.Axml.ViewModelBase be used instead?

**Reply by rockfordlhotka** [reply]
That totally sounds like a bug!
